lightwood/encoder/audio/mfcc.py

```diff
@@ -9,20 +9,40 @@
 
 class MFCCEncoder(BaseEncoder):
     """
-    Audio encoder. Uses `librosa` to compute the Mel-frequency spectral coefficients (MFCCs) of the audio file. They are a common feature used in speech and audio processing. The features are a 2D array, flattened into a 1D one.
+    Audio encoder.
```

Comment:
ive been putting my docstrings in the init; since rst will auto-type arguments, i dont know if init is automatically recognized if the docstring is up here (hence i dunno if the params will be shown).

it's very possible that's the case - if so, I will move my docstrings up under the class definition, otherwise, let's put it in the init call.